programs/test/udp_proxy.c

```diff
@@ -604,6 +706,7 @@ int main( int argc, char *argv[] )
 {
     int ret = 1;
     int exit_code = MBEDTLS_EXIT_FAILURE;
+    uint8_t delay_idx;
```

Comment:
It's a cosmetic thing, but unless we compare against an unsigned value, I don't think there is a reason to use an unsigned types as a loop index.